prisma/schema.prisma

```diff
@@ -157,20 +170,21 @@ model group_target {
 //
 
 model question {
-  id                          Int                          @id(map: "PK_21e5786aa0ea704ae185a79b2d5") @default(autoincrement())
-  createdById                 Int
-  title                       String
-  content                     String
-  type                        Int
-  groupId                     Int?
-  createdAt                   DateTime                     @default(now()) @db.Timestamp(6)
-  updatedAt                   DateTime                     @default(now()) @db.Timestamp(6)
-  deletedAt                   DateTime?                    @db.Timestamp(6)
-  group_question_relationship group_question_relationship?
-  user                        user                         @relation(fields: [createdById], references: [id], onDelete: NoAction, onUpdate: NoAction, map: "fk_question_created_by_id")
-  question_follower_relation  question_follower_relation[]
-  question_query_log          question_query_log[]
-  question_topic_relation     question_topic_relation[]
+  id                            Int                            @id(map: "PK_21e5786aa0ea704ae185a79b2d5") @default(autoincrement())
```

Comment:
It seems that uniform lr endline setting does not work...

Reply:
Maybe I don't know how to apply these settings, it looks like it is not applied automatically.

Reply:
No. The git record is caused by the automatic indentation of prisma. One line, ```QuestionElasticsearchRelation QuestionElasticsearchRelation?```, is added, and its first column is longer than other columns, so the indentation changed.